flutter/lib/device_info.dart

```diff
@@ -63,10 +63,10 @@ class DeviceInfo {
     final deviceNames = DeviceMarketingNames();
 
     return DeviceInfo(
-      manufacturer: deviceInfo.manufacturer,
-      modelCode: deviceInfo.model,
+      manufacturer: deviceInfo.manufacturer ?? 'Unknown',
+      modelCode: deviceInfo.model ?? 'Unknown',
```

Comment:
I think it would be better to use an empty string instead of unknown, and then transform it to `unknown` or any other value in UI.
Checking for an empty string would be easier and less error-prone than comparing to `Unknown` if we want to do any processing of missing values in the future.

Reply:
Empty string is convenient but I try to avoid using it when we have nullable type.
I think we can make the `manufacturer` and `modelCode` nullable, and keep it nullable until we really need to convert it to a string (`""` or `"unknow"`) like showing in the UI or export to JSON?

Reply:
Yeah, making it nullable makes sense.

Reply:
Done